bitmap-ram-rod/lib/transformer.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+const transform = module.exports = {};
+
+transform.modify = (parent, blueCallback, greenCallback, redCallback) => {
+
+  const helper = (bmp, position) => (callback) => (...ind) => {
```

Comment:
I applied the helper to clean up the callback calls in the for loop. Originally it was a lot of content per line which made readability difficult. Now that I look at it, I definitely could consolidate the helper function into a single stack. That's how it was originally.